itou/siae_evaluations/models.py

```diff
@@ -763,6 +784,11 @@ def can_upload(self):
             and self.evaluated_job_application.evaluated_siae.reviewed_at
         )
 
+    def review_state_for_siae(self):
+        if self.evaluated_job_application.hide_state_from_siae():
+            return evaluation_enums.EvaluatedAdministrativeCriteriaState.PENDING
```

Comment:
Là un document déjà accepté en phase 2/2bis réapparaîtra comme `PENDING` en phase 3/3bis après la soumission en phase contradictoire d'autres documents on est d'accord ?

Reply:
Nope. C’est plus bancal que ça.
L’utilisation de l’état de l’`EvaluatedAdministrativeCriteria` n’est que pour voir s’il est dans l’état accepted ou refused et afficher le badge “validé” ou “refusé”.
https://github.com/betagouv/itou/blob/797661ac53f51f6e992bb489d0b553ffe09ef16e/itou/templates/siae_evaluations/includes/criterion_infos.html#L6-L14

Donc, l’affichage est correct. Mais en effet, la méthode ne sert à rien. J’aurais dû la retirer en même temps que j’ai rajouté la règle pour la phase 3bis (`any(crit.submitted_at > reviewed_at for crit in ...)`).

Reply:
Mais donc en phase 3/3bis, la SIAE aura potentiellement accès aux résultats de ses critères via [includes/criterion_infos.html](https://github.com/betagouv/itou/blob/797661ac53f51f6e992bb489d0b553ffe09ef16e/itou/templates/siae_evaluations/includes/criterion_infos.html#L6-L14) et cela avant la fin de la phase 3bis donc avant qu'ils ne soient officiels ?

Reply:
Exact. C’est pour ça que j’avais fait la première version (puis oublié). Au moins cette fois, j’ajoute un test pour qu’on se souvienne !

Reply:
Heu... non. C’est couvert par https://github.com/betagouv/itou/blob/379b2bcfa8d267155b1279ba5785a05edc05abc3/tests/www/siae_evaluations_views/test_siaes_views.py#L198-L249.

Reply:
Si, tu as raison. Le badge de la candidature est bien testé, mais pas le détail des justificatifs. :+1:

Reply:
J’ai mis à jour les tests pour vérifier la présence de cet indicateur et donc couvrir la méthode `EvaluatedAdministrativeCriteria.review_state_for_siae()` qui avait été retirée.